spec/support/mockers/category.rb

```diff
@@ -0,0 +1,20 @@
+require Rails.root.join('spec/support/mockers/mocker.rb')
+require Rails.root.join('spec/support/mockers/equipment_model.rb')
+
+class CategoryMock < Mocker
```

Comment:
... Shouldn't the filename match the class name, just as a matter of convention? This is true for all of the `*Mock` classes.